### PR TITLE
New Source: Zoom [low-code CDK]

### PR DESCRIPTION
**Pull request by subhamX**
## What
The following PR adds the new low code CDK-based Zoom Connector. For more context, please refer to [this issue](https://github.com/airbytehq/connector-contest/issues/111).
https://github.com/airbytehq/connector-contest/issues/111
## How
Our overall aim is to replace the current Zoom connector, which uses [singer-io/tap-zoom](https://github.com/singer-io/tap-zoom/). So in this first phase, this connector implements all the APIs which were earlier part of tap-zoom using the low-code CDK. 
> Please note that [singer-io/tap-zoom](https://github.com/singer-io/tap-zoom/) is no longer in sync with the latest changes to the Zoom API schema. This new connector makes necessary breaking changes in some instances.

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

## Tests

<details><summary><strong>Integration</strong></summary>

image

```
(.venv) @subhamX ➜ .../airbyte/airbyte-integrations/connectors/source-zoom (feat/zoom-connector-using-low-code-cdk) $ python -m pytest integration_tests -p integration_tests.acceptance
Test session starts (platform: linux, Python 3.10.4, pytest 6.2.5, pytest-sugar 0.9.5)
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/workspaces/airbyte/airbyte-integrations/connectors/source-zoom/.hypothesis/examples')
rootdir: /workspaces/airbyte, configfile: pytest.ini
plugins: sugar-0.9.5, requests-mock-1.9.3, timeout-1.4.2, cov-3.0.0, mock-3.6.1, hypothesis-6.54.6
collecting ... 
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_config_match_spec[inputs0] ✓                                       4% ▍         
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_match_expected[inputs0] ✓                                          8% ▊         
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_docker_env[inputs0] ✓                                             12% █▎        
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_oneof_usage[inputs0] ✓                                            15% █▋        
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_required[inputs0] ✓                                               19% █▉        
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_optional[inputs0] ✓                                               23% ██▍       
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_has_secret[inputs0] ✓                                             27% ██▊       
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_secret_never_in_the_output[inputs0] ✓                             31% ███▏      
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_defined_refs_exist_in_json_spec_file[inputs0] ✓                   35% ███▌      
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_oauth_flow_parameters[inputs0] ✓                                  38% ███▉      Pulling docker image airbyte/source-zoom:latest
{"type": "LOG", "log": {"level": "WARN", "message": "\n We did not find the airbyte/source-zoom:latest image for this connector. This probably means this version has not yet been published to an accessible docker registry like DockerHub."}}

 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_backward_compatibility[inputs0] s                                 42% ████▎     
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestSpec.test_additional_properties_is_true[inputs0] ✓                          46% ████▋     
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestConnection.test_check[inputs0] ✓                                            50% █████     
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestConnection.test_check[inputs1] ✓                                            54% █████▍    
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_discover[inputs0] ✓                                          58% █████▊    
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_defined_cursors_exist_in_schema[inputs0] ✓                   62% ██████▎   
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_defined_refs_exist_in_schema[inputs0] ✓                      65% ██████▋   
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_defined_keyword_exist_in_schema[inputs0-allOf] ✓             69% ██████▉   
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_defined_keyword_exist_in_schema[inputs0-not] ✓               73% ███████▍  
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_primary_keys_exist_in_schema[inputs0] ✓                      77% ███████▊  
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_streams_has_sync_modes[inputs0] ✓                            81% ████████▏ 
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_additional_properties_is_true[inputs0] ✓                     85% ████████▌ Pulling docker image airbyte/source-zoom:latest
{"type": "LOG", "log": {"level": "WARN", "message": "\n We did not find the airbyte/source-zoom:latest image for this connector. This probably means this version has not yet been published to an accessible docker registry like DockerHub."}}

 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestDiscovery.test_backward_compatibility[inputs0] s                            88% ████████▉ 
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestBasicRead.test_read[inputs0] ✓                                              92% █████████▎{"type": "LOG", "log": {"level": "ERROR", "message": "Docker container was failed, code 1, error:\n{\"type\": \"TRACE\", \"trace\": {\"type\": \"ERROR\", \"emitted_at\": 1666244812684.525, \"error\": {\"message\": \"Something went wrong in the connector. See the logs for more details.\", \"internal_message\": \"2 validation errors for ConfiguredAirbyteCatalog\\nstreams -> 0 -> sync_mode\\n  value is not a valid enumeration member; permitted: 'full_refresh', 'incremental' (type=type_error.enum; enum_values=[<SyncMode.full_refresh: 'full_refresh'>, <SyncMode.incremental: 'incremental'>])\\nstreams -> 0 -> destination_sync_mode\\n  value is not a valid enumeration member; permitted: 'append', 'overwrite', 'append_dedup' (type=type_error.enum; enum_values=[<DestinationSyncMode.append: 'append'>, <DestinationSyncMode.overwrite: 'overwrite'>, <DestinationSyncMode.append_dedup: 'append_dedup'>])\", \"stack_trace\": \"Traceback (most recent call last):\\n  File \\\"/airbyte/integration_code/main.py\\\", line 13, in <module>\\n    launch(source, sys.argv[1:])\\n  File \\\"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\\\", line 123, in launch\\n    for message in source_entrypoint.run(parsed_args):\\n  File \\\"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\\\", line 111, in run\\n    config_catalog = self.source.read_catalog(parsed_args.catalog)\\n  File \\\"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/source.py\\\", line 90, in read_catalog\\n    return ConfiguredAirbyteCatalog.parse_obj(self.read_config(catalog_path))\\n  File \\\"pydantic/main.py\\\", line 521, in pydantic.main.BaseModel.parse_obj\\n  File \\\"pydantic/main.py\\\", line 341, in pydantic.main.BaseModel.__init__\\npydantic.error_wrappers.ValidationError: 2 validation errors for ConfiguredAirbyteCatalog\\nstreams -> 0 -> sync_mode\\n  value is not a valid enumeration member; permitted: 'full_refresh', 'incremental' (type=type_error.enum; enum_values=[<SyncMode.full_refresh: 'full_refresh'>, <SyncMode.incremental: 'incremental'>])\\nstreams -> 0 -> destination_sync_mode\\n  value is not a valid enumeration member; permitted: 'append', 'overwrite', 'append_dedup' (type=type_error.enum; enum_values=[<DestinationSyncMode.append: 'append'>, <DestinationSyncMode.overwrite: 'overwrite'>, <DestinationSyncMode.append_dedup: 'append_dedup'>])\\n\", \"failure_type\": \"system_error\"}}}\n"}}

 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py::TestBasicRead.test_airbyte_trace_message_on_failure[inputs0] ✓                  96% █████████▋
 airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_full_refresh.py::TestFullRefresh.test_sequential_reads[inputs0] ✓                       100% ██████████
{"type": "LOG", "log": {"level": "INFO", "message": "/workspaces/airbyte/airbyte-integrations/connectors/source-zoom - SAT run - eb30e0aa9b98c4c09e9268c5f903d3ce9aa1a28b - PASSED"}}

================================================================================= short test summary info ==================================================================================
SKIPPED [1] ../../bases/source-acceptance-test/source_acceptance_test/plugin.py:60: Skipping TestIncremental.test_two_sequential_reads because not found in the config
SKIPPED [1] ../../bases/source-acceptance-test/source_acceptance_test/tests/test_core.py:51: The previous connector image could not be retrieved.
SKIPPED [1] ../../bases/source-acceptance-test/source_acceptance_test/tests/test_core.py:229: The previous connector image could not be retrieved.

Results (384.04s):
      24 passed
       2 skipped
(.venv) @subhamX ➜ .../airbyte/airbyte-integrations/connectors/source-zoom (feat/zoom-connector-using-low-code-cdk) $ 
```

</details>

<details><summary><strong>Acceptance</strong></summary>

image

```
@subhamX ➜ .../airbyte/airbyte-integrations/connectors/source-zoom (feat/zoom-connector-using-low-code-cdk) $ ./acceptance-test-docker.sh 
[+] Building 43.7s (17/17) FINISHED                                                                                                                        
 => [internal] load .dockerignore                                                                                                                     0.4s
 => => transferring context: 94B                                                                                                                      0.0s
 => [internal] load build definition from Dockerfile                                                                                                  0.3s
 => => transferring dockerfile: 1.08kB                                                                                                                0.1s
 => [internal] load metadata for docker.io/library/python:3.9.11-alpine3.15                                                                           3.7s
 => [auth] library/python:pull token for registry-1.docker.io                                                                                         0.0s
 => [internal] load build context                                                                                                                     0.2s
 => => transferring context: 65.97kB                                                                                                                  0.0s
 => [base 1/1] FROM docker.io/library/python:3.9.11-alpine3.15@sha256:45ddd216e6b4efee0617e15d541e9148ffd6898203fcbe86a9f5bf906ce7837f                4.0s
 => => resolve docker.io/library/python:3.9.11-alpine3.15@sha256:45ddd216e6b4efee0617e15d541e9148ffd6898203fcbe86a9f5bf906ce7837f                     0.1s
 => => sha256:45ddd216e6b4efee0617e15d541e9148ffd6898203fcbe86a9f5bf906ce7837f 1.65kB / 1.65kB                                                        0.0s
 => => sha256:877ba7355d5c33eec8c6f86ee9151e1ba40f998f49162a5fc056dd97dba033ba 1.37kB / 1.37kB                                                        0.0s
 => => sha256:df5be861e65d2cc84834638758623804bd8781b88f5d9f4deb86a4d6284ab39b 7.05kB / 7.05kB                                                        0.0s
 => => sha256:2807c8f8283e3d8a03430447cb165281f412ee091d2914263420b53bb3e811bd 12.05MB / 12.05MB                                                      0.5s
 => => sha256:c376a57d64541d4fb26bf2ef05f4e63d8ab5428d4ba892cfd034814d8b016934 667.01kB / 667.01kB                                                    0.6s
 => => sha256:3aa4d0bbde192bfaba75f2d124d8cf2e6de452ae03e55d54105e46b06eb8127e 2.81MB / 2.81MB                                                        0.9s
 => => sha256:bb1f4093de6d91f531f596066f0e401a227dff121e113e7d32e503e30c58a0d1 233B / 233B                                                            1.4s
 => => sha256:609deceec33cefeee19d09914055b4476b0bd93e48a60a1aec8ee264295265e3 2.87MB / 2.87MB                                                        1.2s
 => => extracting sha256:3aa4d0bbde192bfaba75f2d124d8cf2e6de452ae03e55d54105e46b06eb8127e                                                             0.5s
 => => extracting sha256:c376a57d64541d4fb26bf2ef05f4e63d8ab5428d4ba892cfd034814d8b016934                                                             0.2s
 => => extracting sha256:2807c8f8283e3d8a03430447cb165281f412ee091d2914263420b53bb3e811bd                                                             0.4s
 => => extracting sha256:bb1f4093de6d91f531f596066f0e401a227dff121e113e7d32e503e30c58a0d1                                                             0.0s
 => => extracting sha256:609deceec33cefeee19d09914055b4476b0bd93e48a60a1aec8ee264295265e3                                                             0.2s
 => [builder 1/4] WORKDIR /airbyte/integration_code                                                                                                   0.2s
 => [builder 2/4] RUN apk --no-cache upgrade     && pip install --upgrade pip     && apk --no-cache add tzdata build-base                             8.5s
 => [builder 3/4] COPY setup.py ./                                                                                                                    0.3s 
 => [builder 4/4] RUN pip install --prefix=/install .                                                                                                21.1s 
 => [stage-2 2/7] COPY --from=builder /install /usr/local                                                                                             0.7s 
 => [stage-2 3/7] COPY --from=builder /usr/share/zoneinfo/Etc/UTC /etc/localtime                                                                      0.2s 
 => [stage-2 4/7] RUN echo "Etc/UTC" > /etc/timezone                                                                                                  0.5s 
 => [stage-2 5/7] RUN apk --no-cache add bash                                                                                                         1.0s 
 => [stage-2 6/7] COPY main.py ./                                                                                                                     0.3s 
 => [stage-2 7/7] COPY source_zoom ./source_zoom                                                                                                      0.3s 
 => exporting to image                                                                                                                                1.6s
 => => exporting layers                                                                                                                               1.6s
 => => writing image sha256:e951edafd2e607b69bdb023564440188f4cd68b441d42b4bfc0c7f517059e2e1                                                          0.0s
 => => naming to docker.io/airbyte/source-zoom:dev                                                                                                    0.0s
latest: Pulling from airbyte/source-acceptance-test
3aa4d0bbde19: Already exists 
c376a57d6454: Already exists 
2807c8f8283e: Already exists 
bb1f4093de6d: Already exists 
609deceec33c: Already exists 
4c67188f50d3: Pull complete 
55d99227321e: Pull complete 
8f5ab48c874d: Pull complete 
6e4dd6f21ff4: Pull complete 
c7854a30af8b: Pull complete 
df4c9dcfe55c: Pull complete 
b998bd40de1f: Pull complete 
bb9d56c6c359: Pull complete 
Digest: sha256:ff2d2d2ae90ed5dfdeeb0a992cb2054bbe279cb6e3d38a4af987cb4f2be606b6
Status: Downloaded newer image for airbyte/source-acceptance-test:latest
docker.io/airbyte/source-acceptance-test:latest
Test session starts (platform: linux, Python 3.9.11, pytest 6.2.5, pytest-sugar 0.9.5)
rootdir: /test_input
plugins: sugar-0.9.5, requests-mock-1.9.3, timeout-1.4.2, cov-3.0.0, mock-3.6.1, hypothesis-6.54.6
collecting ... 
 test_core.py ✓✓✓✓✓✓✓✓✓✓s✓✓✓✓✓✓✓✓✓✓✓s✓✓                                                                                                      96% █████████▋
 test_full_refresh.py ✓                                                                                                                     100% ██████████

================================================================= short test summary info =================================================================
SKIPPED [1] source_acceptance_test/plugin.py:60: Skipping TestIncremental.test_two_sequential_reads because not found in the config
SKIPPED [1] source_acceptance_test/tests/test_core.py:51: The previous connector image could not be retrieved.
SKIPPED [1] source_acceptance_test/tests/test_core.py:229: The previous connector image could not be retrieved.

Results (385.23s):
      24 passed
       2 skipped
@subhamX ➜ .../airbyte/airbyte-integrations/connectors/source-zoom (feat/zoom-connector-using-low-code-cdk) $ 
```

</details>

### CODE REVIEW

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml [hunk 1]
```diff
@@ -1264,3 +1264,10 @@
   documentationUrl: https://docs.airbyte.com/integrations/sources/yandex-metrica
   sourceType: api
   releaseStage: alpha
+- name: ZoomHello
```
Comment (marcosmarxm) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:1267]: Correct here.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
a few comments

File: airbyte-integrations/connectors/source-zoom/acceptance-test-config.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        - "webinar_absentees"
+        - "webinar_polls"
+        - "webinar_poll_results"
+        - "webinar_registration_questions"
+        - "webinar_tracking_sources"
+        - "webinar_qna_results"
+        - "report_meetings"
+        - "report_meeting_participants"
+        - "report_webinars"
+        - "report_webinar_participants"
```
[the remark this answers is not in the document]
Comment (subhamX) [reply airbyte-integrations/connectors/source-zoom/acceptance-test-config.yml:36]: All of them can be empty depending on our account credentials.

**For example:** if you create a new account without meetings, then all meeting streams will be empty.
All webinars will be empty if you're not a PRO member or haven't yet created a webinar.

So I thought I should add all of them here. 🙂

File: airbyte-integrations/connectors/source-zoom/source_zoom/schemas/TODO.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# TODO: Define your stream schemas
```
[the remark this answers is not in the document]
Comment (subhamX) [reply airbyte-integrations/connectors/source-zoom/source_zoom/schemas/TODO.md:1]: Done. Thank you.

File: airbyte-integrations/connectors/source-zoom/integration_tests/catalog.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
+    {
+      "name": "TODO fix this file",
+      "supported_sync_modes": ["full_refresh", "incremental"],
+      "source_defined_cursor": true,
```
[the remark this answers is not in the document]
Comment (subhamX) [reply airbyte-integrations/connectors/source-zoom/integration_tests/catalog.json:6]: Done. Thank you.

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml (hunk 1 above)
Comment (subhamX) [reply airbyte-config/init/src/main/resources/seed/source_definitions.yaml:1267]: Renaming it as "Zoom"./

File: airbyte-integrations/connectors/source-zoom/acceptance-test-config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-zoom/acceptance-test-config.yml:36]: no, this is for testing :)

File: docs/integrations/sources/zoom-lc-cdk.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+# Zoom
+
+## Overview
+
+
+The following connector allows airbyte users to fetch various meetings & webinar data points from the [Zoom](https://zoom.us) source. This connector is built entirely using the [low-code CDK](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview/).
+
```
Comment (marcosmarxm) [inline docs/integrations/sources/zoom-lc-cdk.md:7]: Please remove the or replace the other Zoom documentation. I'll remove the singer code in the future.

File: docs/integrations/README.md [hunk 6]
```diff
@@ -180,6 +180,7 @@ For more information about the grading system, see [Product Release Stages](http
 | [Zenloop](sources/zenloop.md)                                                               | Alpha                | Yes                 |
 | [Zoho CRM](sources/zoho-crm.md)                                                             | Alpha                | No                  |
 | [Zoom](sources/zoom.md)                                                                     | Alpha                | No                  |
+| [Zoom low-code CDK](sources/zoom-lc-cdk.md)                                                        | Alpha                | No                  |
```
Comment (marcosmarxm) [inline docs/integrations/README.md:183]: you can remove this line in favor of the other one Zoom.

File: airbyte-integrations/connectors/source-zoom/acceptance-test-config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (subhamX) [reply airbyte-integrations/connectors/source-zoom/acceptance-test-config.yml:36]: Yes, but for testing if you use a non PRO account then most of them will be empty; 

Are you suggesting that I remove all of them?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: docs/integrations/README.md (hunk 6 above)
Comment (subhamX) [reply docs/integrations/README.md]: Done. Thank you.

File: docs/integrations/sources/zoom-lc-cdk.md (hunk 5 above)
Comment (subhamX) [reply docs/integrations/sources/zoom-lc-cdk.md]: Done. Thank you.

File: airbyte-integrations/connectors/source-zoom/acceptance-test-config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (subhamX) [reply airbyte-integrations/connectors/source-zoom/acceptance-test-config.yml]: Sorry for the delay; I was a bit under the weather; I think you've already made the necessary changes @marcosmarxm; Feel free to mark it resolved. Thank you. 🤗

**Review by marcosmarxm — APPROVED** [review]
Thanks for the contribution @subhamX

**Pull request merged**